app/models/user.rb

```diff
@@ -15,8 +16,13 @@ class User < ApplicationRecord
   validates :first_name, length: { minimum: 3, maximum: 50 }, presence: true
   validates :last_name,  length: { minimum: 3, maximum: 50 }, presence: true
   validates :tenant, presence: true
+  validates :role, presence: true, inclusion: ROLES
```

Comment:
does this really work?
suppose it should be `inclusion: { in: %w(small medium large)`